tools/sof_perf_analyzer.py

```diff
@@ -182,11 +182,19 @@ def process_kmsg_file():
     [  334.818442] kernel: snd_sof:sof_ipc4_log_header: sof-audio-pci-intel-mtl 0000:00:1f.3:
     ipc tx      : 0x40000004|0x15: MOD_INIT_INSTANCE [data size: 84]
 
+    However, for some cases, the above lines does not contain widget ID information, in this
```

Comment:
Which cases? And what exactly is missing in these cases and why? An entire line and if yes which one? A part of a line and if yes which part? You must be much more specific, otherwise it's impossible to review the code without knowing what it is trying to parse.

Reply:
https://github.com/thesofproject/sof-test/issues/1108

Reply:
Thanks, I had heard about 1108 but besides for extraordinary complex situations anyone with basic Python knowledge should be able to understand the code without opening bugs which are not even linked from the source. This code is not an extraordinary complex situation, it's just parsing some logs so it must be easy to follow. Thanks for adding an example!

```suggestion
However, for some cases, the line after "Creating" does not contain widget ID information, in this
```

Also explain why the line after "Creating" does not always contain widget ID information, let's not "program by coincidence".